static/reactbkk.svg

```diff
@@ -0,0 +1 @@
48.673 19.846-18.387 16.055-37.418 40.132-53.348 67.846zm-9.291-5.312c16.49-28.689 36.25-53.69 55.585-70.571 23.619-20.621 46.959-29.23 61.1-21.049 14.736 8.525 19.09 34.655 12.414 67.173-4.885 23.788-16.46 52.124-32.605 80.213-16.944 29.477-35.694 54.057-53.837 69.968-24.054 21.095-48.71 29.832-63.275 21.405-14.073-8.143-18.534-32.11-12.805-62.38 4.792-25.323 16.729-55.717 33.423-84.76z" fill="#00D8FF" fill-rule="nonzero"/><path d="M99.003 145.03c16.135 28.052 35.634 52.694 54.097 68.684 20.152 17.454 38.895 24.091 47.813 18.895 9.405-5.48 13.355-26.91 7.955-54.365-4.412-22.436-15.813-49.986-32.224-78.516-15.622-27.16-33.582-50.502-50.713-65.81-21.608-19.311-42.163-27.034-51.62-21.523-8.913 5.193-12.39 25.5-7.175 52.319 4.67 24.02 15.93 52.608 31.867 80.315zm-9.259 5.331C73.248 121.682 61.558 92.004 56.65 66.755c-5.993-30.828-1.788-55.398 12.285-63.597 14.705-8.568 39.414.715 64.112 22.787 18.06 16.14 36.703 40.369 52.856 68.451 16.949 29.466 28.78 58.057 33.446 81.784 6.185 31.444 1.433 57.221-13.06 65.666-14.045 8.183-36.945.074-60.18-20.05-19.428-16.826-39.664-42.4-56.365-71.435z" fill="#00D8FF" fill-rule="nonzero"/><path d="M138 136c7.18 0 13-5.82 13-13s-5.82-13-13-13-13 5.82-13 13 5.82 13 13 13zm.25-2.5c5.66 0 10.25-4.59 10.25-10.25S143.91 113 138.25 113 128 117.59 128 123.25s4.59 10.25 10.25 10.25zm0-3.5a6.75 6.75 0 1 0 0-13.5 6.75 6.75 0 0 0 0 13.5z" fill="#00D8FF"/><g transform="translate(124 84)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><use fill="#00D8FF" xlink:href="#a"/><path d="M22.496 56.084l.704 3.128 1.152-.704.195.255c1.091 1.712 1.881 3.584 1.881 5.434 0 7.375-5.916 13.354-13.214 13.354S0 71.572 0 64.197c0-1.86.729-3.731 1.738-5.434l.337-.471 1.347.673.658-3.424a35.061 35.061 0 0 1 3.262-3.31c1.552-1.388 5.872-7.037 5.872-7.037s4.066 5.65 5.079 6.671c.464.468 2.207 1.927 3.962 3.905l.24.314zM13.219 71.73c3.937 0 7.128-3.214 7.128-7.18 0-2.986-3.842-6.08-4.388-6.63-.547-.549-2.74-3.586-2.74-3.586s-2.33 3.037-3.168 3.783c-.837.746-3.96 3.615-3.96 6.434 0 3.965 3.19 7.18 7.128 7.18z" fill="#00D8FF" mask="url(#b)" transform="rotate(-180 13.214 61.372)"/></g><use fill="#00D8FF" xlink:href="#c" transform="rotate(90 33.5 143)"/><g transform="rotate(180 75.5 80.5)"><mask id="e" fill="#fff"><use xlink:href="#d"/></mask><use fill="#00D8FF" xlink:href="#d"/><path d="M22.496 56.084l.704 3.128 1.152-.704.195.255c1.091 1.712 1.881 3.584 1.881 5.434 0 7.375-5.916 13.354-13.214 13.354S0 71.572 0 64.197c0-1.86.729-3.731 1.738-5.434l.337-.471 1.347.673.658-3.424a35.061 35.061 0 0 1 3.262-3.31c1.552-1.388 5.872-7.037 5.872-7.037s4.066 5.65 5.079 6.671c.464.468 2.207 1.927 3.962 3.905l.24.314zM13.219 71.73c3.937 0 7.128-3.214 7.128-7.18 0-2.986-3.842-6.08-4.388-6.63-.547-.549-2.74-3.586-2.74-3.586s-2.33 3.037-3.168 3.783c-.837.746-3.96 3.615-3.96 6.434 0 3.965 3.19 7.18 7.128 7.18z" fill="#00D8FF" mask="url(#e)" transform="rotate(-180 13.214 61.372)"/></g><use fill="#00D8FF" xlink:href="#f" transform="rotate(-90 117.5 19)"/></g></svg>
```

Comment:
Please do a find-and-replace `#00D8FF` into `#FFFFFF` in this file, maybe rename it to `reactbkk-white.svg`, so that we can avoid using CSS filters.